src/AWS.DistributedCacheProvider/AWS.DistributedCacheProvider.csproj

```diff
@@ -29,11 +29,11 @@
 	</ItemGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="6.0.0" />
-		<PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="6.0.0" />
-		<PackageReference Include="Microsoft.Extensions.Options" Version="6.0.0" />
-		<PackageReference Include="AWSSDK.DynamoDBv2" Version="3.7.102.11" />
-		<PackageReference Include="AWSSDK.Extensions.NETCore.Setup" Version="3.7.5" />
+		<PackageReference Include="Microsoft.Extensions.Caching.Abstractions" Version="7.0.0" />
```

Comment:
Since we are targeting .NET 6 I think are minimum for these `Microsoft.Extensions` packages should be `6.0.0`.

Reply:
I've never understood our convention - why is that? https://www.nuget.org/packages/Microsoft.Extensions.Caching.Abstractions/7.0.0#supportedframeworks-body-tab still targets .NET 6 explicitly.

Reply:
By using the .NET 7 versions if the client application is targeting .NET 6 and you do `dotnet publish` on the project you can see the version 7 dlls are copied into the publish folder. But by using the .NET 6 when you do the `dotnet publish` they are not copied over because it knows it can just use the versions that are part of the runtime. So we are increasing the deployment bundle and forcing a new version over the runtime.

If the client application is targeting .NET 7+ and our library is referencing .NET 6 then we are still not copying over our versions and the runtime's version is being used. 

So basically by matching the versions of the `Microsoft.Extension.*` packages to our minimum target framework we are making it so we will just use the runtime's versions of the assembly and not add to the deployment bundle size.

Reply:
Thanks for the explanation, makes sense. Updates this to the latest `6.0.x`.